configure.ac

```diff
@@ -1501,9 +1501,7 @@ else
    AS_CASE([$host], [*-*-cygwin], [have_hwloc=no])
 
    if test "$have_hwloc" = "yes" ; then
-      hwloclib="-lhwloc"
       if test -d ${with_hwloc_prefix}/lib64 ; then
-         PAC_APPEND_FLAG([-L${with_hwloc_prefix}/lib64],[WRAPPER_LDFLAGS])
```

Comment:
What's this line for?

Reply:
Should `AC_ARG_WITH` above be replace with `PAC_CHECK_PREFIX([hwloc])` like it is done in Hydra? That way CPPFLAGS and LDFLAGS are set consistently and this part (setting hwloclibdir) can just be deleted.